exercises/high-scores/.meta/src/reference/java/HighScores.java

```diff
@@ -0,0 +1,32 @@
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.OptionalInt;
+
+import static java.util.Arrays.copyOf;
+import static java.util.Arrays.sort;
+
+class HighScores {
+    private int[] highScores;
+    public HighScores(int[] highScores) {
+        this.highScores = highScores;
```

Comment:
Best practice: make a defensive copy.

```suggestion
        this.highScores = highScores.clone();
```